luarules/gadgets/ai/shard_runtime/spring_lua/unit.lua

```diff
@@ -615,7 +615,6 @@ end
 
 function ShardUnit:GetPosition()
 	local bpx, bpy, bpz = Spring.GetUnitPosition(self.id)
-	local isDead = Spring.GetUnitIsDead(self.id)
 	--[[if isDead or isDead == nil then
```

Comment:
Looks like `isDead` was "used" in this commented-out chunk below, remove that chunk as well since else it will break if somebody uncomments it.